integration/benchmark/contLoadGenerator.go

```diff
@@ -719,26 +719,25 @@ func (lg *ContLoadGenerator) sendTokenTransferTx(workerID int) {
 		return
 	}
 
-	log.Trace().Hex("txID", transferTx.ID().Bytes()).Msg("transaction sent")
+	log = log.With().Hex("tx_id", transferTx.ID().Bytes()).Logger()
+	log.Trace().Msg("transaction sent")
 
-	for {
-		select {
-		case <-ch:
-			log.Trace().
-				Hex("txID", transferTx.ID().Bytes()).
-				Dur("duration", time.Since(startTime)).
-				Msg("transaction confirmed")
-			lg.workerStatsTracker.IncTxExecuted()
-			return
-		case <-time.After(slowTransactionThreshold):
-			// TODO(rbtz): add a counter for slow transactions
-			log.Trace().
-				Hex("txID", transferTx.ID().Bytes()).
-				Dur("duration", time.Since(startTime)).
-				Int("availableAccounts", len(lg.availableAccounts)).
-				Msg("is taking too long")
-		}
+	t := time.NewTimer(lostTransactionThreshold)
```

Comment:
just use time.After?  you're not leaking anything

Reply:
Timeout for the After is quite big, so I've decided to release resources immediately if possible.